app/src/full/java/io/homeassistant/companion/android/location/HighAccuracyLocationService.kt

```diff
@@ -125,7 +125,7 @@ class HighAccuracyLocationService : Service() {
             val disablePendingIntent = PendingIntent.getBroadcast(context, 0, disableIntent, PendingIntent.FLAG_MUTABLE)
 
             notificationBuilder = NotificationCompat.Builder(context, channelID)
-                .setSmallIcon(R.drawable.ic_stat_ic_notification)
+                .setSmallIcon(io.homeassistant.companion.android.common.R.drawable.ic_stat_ic_notification)
```

Comment:
I'm not too keen on this, but the build works. Had to move some resources to the `common` module but you have to reference the resources from the `common` module's `R` file. Any idea how to refererence better? Some options:
a) add `common/main/res` to `app` and `wear` sourcesets
b) maybe use `ids.xml` to reference the library resources as if it's in their own `res` directory
c) use import alias (should be an implementation detail and something that maintainers should need to worry about)

Reply:
I opted for b) but didn't need the package name in the end. Hope it helps in the future